src/styles/index.scss

```diff
@@ -1,3 +1,37 @@
+@import './utils/variables';
+@import './blocks/header';
+@import './blocks/card';
+@import './blocks/stars';
+
 body {
   margin: 0;
+  box-sizing: border-box;
+  display: block;
+}
+
+p {
+  margin: 0;
+}
+
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 48px 46px;
+  padding: 110px 40px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 1fr);
+  }
+
+  @media (min-width: 1200px) {
+    gap: 46px 48px;
+  }
 }
```

Comment:
The .catalog styles should be moved to a separate file in the blocks folder. This is because .catalog is a block according to BEM methodology, and each block should have its own file.